src/backend/test/utils/valid-upgrade-candidates/v100RC_v100RC2.json

```diff
@@ -0,0 +1,30 @@
+{
+  "currentApkInfo": {
+    "hash": "810ff2fb252a5dee4220f2cb0e6a519891fb67f2f828a6cab4ef8894633b1f50",
+    "size": 8,
+    "versionName": "1.1.0-RC+9a6731e",
+    "minSdkVersion": 1,
+    "applicationId": "com.example",
+    "versionCode": 1,
+    "hashType": "sha256",
+    "platform": "android",
+    "arch": ["arm64-v8a"],
+    "filepath": "non-existent.apk"
+  },
+  "deviceInfo": {
+    "supportedAbis": ["arm64-v8a"],
+    "sdkVersion": 1
+  },
+  "installer": {
+    "hash": "810ff2fb242a5dee4220f2cb0e6a519891fb67f2f828a6cab4ef8894633b1f50",
+    "size": 8,
+    "versionName": " 1.1.0-RC+19d986e",
+    "minSdkVersion": 1,
+    "applicationId": "com.example",
+    "versionCode": 1,
```

Comment:
This is not a valid upgrade candidate, because although it comes from the same RC (`1.1.0-RC`) and it is different (because the commit hash is different) the version code is the same. We should only count as valid with a higher version code, otherwise we would get in a loop back and forth.